h/groups/test/views_test.py

```diff
@@ -155,6 +155,20 @@ def test_create_with_non_ascii_name():
     views.create(_mock_request(params={"name": u"☆ ßüper Gröup ☆"}))
 
 
+@create_fixtures
+def test_create_broadcasts_join_event(Group):
```

Comment:
Minor niggle, but `broadcasts` is a bit misleading here, I think. `publishes` might be a bit clearer. I'm guessing this is a hangover from when it was in `h.api`.